addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/thermostat/NATherm1Handler.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.handler.thermostat;
+
+import static org.openhab.binding.netatmo.NetatmoBindingConstants.*;
+
+import java.util.Calendar;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.netatmo.handler.NetatmoModuleHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.swagger.client.model.NADevice;
+import io.swagger.client.model.NADeviceListBody;
+import io.swagger.client.model.NAModule;
+import io.swagger.client.model.NAThermStateBody;
+import io.swagger.client.model.NAThermStateResponse;
+
+/**
+ * {@link NATherm1Handler} is the class used to handle the thermostat
+ * module of a thermostat set
+ *
+ * @author Gaël L'hopital - Initial contribution OH2 version
+ *
+ */
+public class NATherm1Handler extends NetatmoModuleHandler {
+    private static Logger logger = LoggerFactory.getLogger(NATherm1Handler.class);
+    private NAThermStateBody thermStateBody;
+    private Integer setpointDefaultDuration = null;
+
+    public NATherm1Handler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    protected void updateChannels() {
+        try {
+            NAThermStateResponse thermState = bridgeHandler.getThermostatApi().getthermstate(getParentId(), getId());
+            thermStateBody = thermState.getBody();
+
+            NADeviceListBody deviceList = bridgeHandler.getThermostatApi().devicelist(actualApp, getParentId(), false)
+                    .getBody();
+            for (NAModule module : deviceList.getModules()) {
+                if (module.getId().equalsIgnoreCase(getId())) {
+                    this.module = module;
+                    super.updateChannels();
+                }
+            }
+        } catch (Exception e) {
+            logger.error(e.getMessage());
```

Comment:
don't log an error, better provide the error message within the Thing status as this will also be shown in the UI.